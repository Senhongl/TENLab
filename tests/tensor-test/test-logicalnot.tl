a = [[1,0,3],[4,0,-1]];
print(!a);