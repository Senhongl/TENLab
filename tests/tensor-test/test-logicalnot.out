 0  1  0
 0  1  0
[ CPUIntType{2,3} ]
